### PR TITLE
Article model added

### PR DESCRIPTION
**Pull request by Erfan2003**

### CODE REVIEW

File: blog/models.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
 from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+class Article(models.Model):
+    title = models.CharField(max_length=220, verbose_name="Title")
+    content = models.TextField(verbose_name="Content")
```
Comment (RealNitroZeus) [inline blog/models.py:11]: Use Ckeditor here

File: blog/models.py [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.DO_NOTHING, related_name="articles"
```
Comment (RealNitroZeus) [inline blog/models.py:13]: Change it to set null

File: blog/models.py [hunk 3]
```diff
@@ -3,12 +3,18 @@
...
+        PUBLISH = "p"
+
     title = models.CharField(max_length=220, verbose_name="Title")
     content = models.TextField(verbose_name="Content")
     author = models.ForeignKey(
         settings.AUTH_USER_MODEL, on_delete=models.DO_NOTHING, related_name="articles"
     )
     date = models.DateTimeField(auto_now_add=True)
+    status = models.CharField(max_length=1, choices=Status.choices)
+    slug = models.SlugField(max_length=1024, verbose_name="Slug")
```
Comment (RealNitroZeus) [inline blog/models.py:17]: lower the max length to 50

**Comment by RealNitroZeus** [comment]
Move it to blog_base branch

**Comment by Erfan2003** [comment]
> FIx mentioned issues in comments

OK

**Pull request closed without merging**